apps/wakunode2/wakunode2.nim

```diff
@@ -47,25 +47,18 @@ import
   ../../waku/v2/protocol/waku_peer_exchange,
   ../../waku/v2/protocol/waku_relay/validators,
   ../../waku/v2/utils/peers,
-  ./wakunode2_setup_rest,
-  ./wakunode2_setup_rpc,
   ./config
 
-when defined(rln):
-  import
-    ../../waku/v2/protocol/waku_rln_relay
-
-
 logScope:
   topics = "wakunode"
 
-type SetupResult[T] = Result[T, string]
-
 # Temporarily merge `app.nim` into `wakunode2.nim` until we encapsulate the
 #  state and the logic in an object instance.
 include ./app
 
 when defined(waku_exp_store_resume):
+  type SetupResult[T] = Result[T, string]
+
   proc resumeMessageStore(node: WakuNode, address: string): Future[SetupResult[void]] {.async.} =
```

Comment:
if resumeMessageStore is using SetupResult , shouldn't it also be wrapped within the `when defined` block?

if not, maybe we shouldn't define the SetupResult type in the `when defined ` block

Reply:
And it is under the `when defined` block 🤔